HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 History
 =======
 
+3.0.5 (2024-11-24)
```

Comment:
Change to 4.0.0 and maybe add a line on that we are deprecating support for python 3.8.

Just checked and also saw that we should probably update the reference to python 3.7 to 3.9 in `.github/workflows/python-publish.yml`

Reply:
Done. Updated both in the commits below.